practical/groovy/crop_rectangle_from_image.groovy

```diff
@@ -0,0 +1,185 @@
+/*
+ * -----------------------------------------------------------------------------
+ *  Copyright (C) 2019 University of Dundee. All rights reserved.
+ *
+ *
+ *  This program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2 of the License, or
+ *  (at your option) any later version.
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * ------------------------------------------------------------------------------
+ */
+
+/*
+ * This Groovy script uses ImageJ to crop an image.
+ * The cropped image is saved locally as OME-TIFF using Bio-Formats Exporter.
+ * Upload the OME-TIFF to OMERO.
```

Comment:
Maybe ``Upload the OME....`` -> ``The OME-TIFF is then uploaded to OMERO.``
?

Reply:
Maybe even better: ``The script then uploads the OME-TIFF to OMERO, to a new dataset named "Cropped".``